components/invest/NftInvest.tsx

```diff
@@ -38,12 +32,18 @@ function NftInvest() {
               display='flex'
               width={'90%'}
               justifyContent='center'>
-              <Button onClick={() => window.open('https://opensea.io/collection/chippi', '_blank')}>Invest in Chippi</Button>
+              <Button
```

Comment:
MINOR: this looks a bit weird, probably legacy code. 

Might be better to use next/link instead